code/software/libs/src/libm/common/acosl.c

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
(Caveat: I've not checked the whole PR): I would like to add this licence text at the top level, along with the other licences (perhaps as `LICENCE.libm` or something).

Reply:
(Also, gonna take the rest of the PR as read, given that I think it's all IBM code? Please correct me if I'm wrong...)

Reply:
(Okay, I see there are numerous licences in here, IBM, ARM, Sun, RedHat. Nothing worrying, but If we could reproduce those licenses at the top level that would be super helpful in terms of reproducing them :) )

Reply:
These are all the companies who've contributed over time to libm. Amazing really, when you think about it. I'll copy them. I've included the newlib licenses.